README.md

```diff
@@ -88,8 +88,7 @@ Please see the dedicated [README.md](tools/docker/README.md) for information on
 
 * [Getting Started](doc/website/getting-started/overview.md)
 * [Installation Guide](doc/website/getting-started/installation.md)
-* [Concepts](doc/conceptual-guide.md)
```

Comment:
Would it make sense to link to `/doc/website/concepts/`?

Reply:
I don't think detailed background on how shared memory works is needed for users. The important article is `overview.md` and is linked two lines above.